### DISCUSSION TITLE
Suggest me a malware / stealer to try inside Sandboxie

### DISCUSSION THREAD

**Discussion by macruspareto** [sandboxie-plus/Sandboxie#2775] [General]
Hello folks !

Would you suggest any proof-of-concept open-src malware / stealer / dumper to try inside my Sandboxie to see what *could* be exfiltrated and what additional restrictions I should apply?

There's a project called 'Stealerium' here on Git, but like most of them, its aimed at stealing actual credentials (messengers, browser cookies, cryptowallets) while I'm more interested in blocking system-wide GUIDs/UUIDs, sensitive registry keys (like encoded serial numbers for commercial software) etc. Thanks !

**Comment by bastik-1001** [comment]
Please note that, by default, the software run under Sandboxie is able to read the same data and keys the user has access to. Historically, there was seemingly no concern to what some program was able to read, just that it could not make changes. This applies to today. The recently introduced privacy mode improves on that, but as far as I can tell isn't the main use-case.

To address the question, I am not aware of something like that being a thing, but others might be able to provide you with some direction regarding some software to test a sandbox against a specific threat model.
